test/e2e/storage/persistent_volumes.go

```diff
@@ -321,7 +321,7 @@ var _ = utils.SIGDescribe("PersistentVolumes", func() {
 	// doesn't have the ntfs parameter, we can't change the status of the cluster
 	// to add a StorageClass that's compatible with windows which is also the
 	// default StorageClass
-	ginkgo.Describe("Default StorageClass [LinuxOnly]", func() {
+	ginkgo.Describe("Default StorageClass [LinuxOnly] [Feature:StorageProviderGCEPD]", func() {
```

Comment:
I think this test can run on any cluster with a default storageclass. We probably want to add the skip method here.

Reply:
done